index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <nav class="container">
+            <h2 class="barItem">Welcome</h2>
+            <img class="barItem" src="images/Icon - Menu.png" alt="Burger Menu Icon">
+        </nav>
+    </header>
+    <section class="tittleContainer">
+        <h1 class="blue">Hey there. I'm Victor<br><span class="green"> I'm a software developer</span></h1>
+    </section>
+    <section class="helperText">
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    </section>
```

Comment:
- [ ] Good job using the `section` tag 🈹, but I think you could use just one tag instead of two, in this case, is semantically correct, further always inside of a `<section >` tag should be a `<hx>` tag, you can check this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more information. Kindly change it. 📏